blocks/api/test/factory.js

```diff
@@ -6,8 +6,9 @@ import { noop } from 'lodash';
 /**
  * Internal dependencies
  */
-import { createBlock, switchToBlockType } from '../factory';
+import { createBlock, switchToBlockType, transformComponentToText } from '../factory';
 import { getBlockTypes, unregisterBlockType, setUnknownTypeHandler, registerBlockType } from '../registration';
+import { Component } from 'element'
```

Comment:
This should be in a separate "WordPress dependencies block":

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#imports

Note with #2172, these need to be updated to prefix the dependencies with `@wordpress/`. You will need to perform a rebase against the latest version of master and apply your changes:

```
git fetch origin
git rebase origin/master
```